.github/workflows/e2e.yaml

```diff
@@ -14,9 +14,9 @@ jobs:
       matrix:
        kube-version:
        - "1.19"
```

Comment:
@jpkrohling any specific reason, why are we dropping `1.20` but not the `1.19` ?

Reply:
I thought we had removed 1.19 already. It should indeed be removed.